manifests/d/dail8859/NotepadNext/0.5.4/dail8859.NotepadNext.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: dail8859.NotepadNext
+PackageVersion: 0.5.4
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/dail8859/NotepadNext/releases/download/v0.5.4/NotepadNext-v0.5.4-Installer.exe
+  InstallerSha256: A45822E2777D4A6848538A0D3102784578FD95815D34F45B3A9DE139C04EB2A6
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/dail8859/NotepadNext/releases/download/v0.5.4/NotepadNext-v0.5.4-Installer.exe
+  InstallerSha256: A45822E2777D4A6848538A0D3102784578FD95815D34F45B3A9DE139C04EB2A6
+  InstallerSwitches:
+    Custom: /ALLUSERS
```

Display Names are recommended here to help with scope
```suggestion
Installers:
- Architecture: x64
  Scope: user
  InstallerUrl: https://github.com/dail8859/NotepadNext/releases/download/v0.5.4/NotepadNext-v0.5.4-Installer.exe
  InstallerSha256: A45822E2777D4A6848538A0D3102784578FD95815D34F45B3A9DE139C04EB2A6
  InstallerSwitches:
    Custom: /CURRENTUSER
  AppsAndFeaturesEntries: 
    - DisplayName: Notepad Next v0.5.4 (current user)
- Architecture: x64
  Scope: machine
  InstallerUrl: https://github.com/dail8859/NotepadNext/releases/download/v0.5.4/NotepadNext-v0.5.4-Installer.exe
  InstallerSha256: A45822E2777D4A6848538A0D3102784578FD95815D34F45B3A9DE139C04EB2A6
  InstallerSwitches:
    Custom: /ALLUSERS
  AppsAndFeaturesEntries: 
    - DisplayName: Notepad Next v0.5.4
```

---

@sitiom Thoughts on this suggestion? Will the WinGet Releaser workflow update the display name appropriately in this case?

---

Or would it be better to not include the version number in the DisplayName, e.g. `Notepad Next (current user)` and `Notepad Next`? Not sure what the best-practice.

---

Vedant's action won't update the AppsAndFeaturesEntries, so leaving the version out of the DisplayName would be helpful, especially since the Control Panel will show the version a few columns over anyways.

---

@Trenly Agreed.